### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by Levminer** [community/community#21792] [New to GitHub]
<p>Hi, I applied for the pack on Feb 14. and I’m still waiting. I contacted the support in email (no response 3days). I contacted the support on Twitter two times (4days, 1day). I know its free, but why is it exist if I can get it?</p>

**Comment by logankilpatrick** [comment]
<p>Hey <a class="mention" href="/levminer">@levminer</a>, because students are manually verified for the program and there are so many students applying, wait times can be extended. Hang tight! </p>

**Comment by logankilpatrick** [comment]
<p>Also, <a class="mention" href="/levminer">@levminer</a>, big fan of the profile picture. The New Tesla Roadster looks awesome!  </p>

**Comment by Levminer** [comment]
<p>Thanks, I’m a big Tesla fan to :slight_smile:</p>

**Comment by Levminer** [comment]
<p>Ok, but why are they advertising it on Twitter and on the blog if they can’t verify the requests? Btw I applied with school e-mail, so its should be an instant verify. And why the support dosen’t answer me? on the website they say “24/7 support”. And what if I get rejected after a month? I have to wait a month again?</p>

**Comment by Levminer** [comment, marked as answer]
<p>I got my pack after a month! Thanks GitHub :slight_smile:</p>
